### ISSUE TITLE
Implicit intersection results in #VALUE

### ISSUE THREAD

**Issue by keithclarkj** [EPPlusSoftware/EPPlus#1239]
When attempting to use an implicit intersection formula, the calculation results in a `#VALUE!` , even after setting the `UseImplicitIntersection` flag to true. This still happens when adding the `@` like in excel.

Behaviour in excel:
![image](https://github.com/EPPlusSoftware/EPPlus/assets/77936136/224846f4-2537-4f06-8391-2b61e8d0751d)

Unit test is below:

```
[TestMethod]
public void ImplicitIntersection_ColumnReference()
{
    using (var pck = new ExcelPackage())
    {
        var sheet1 = pck.Workbook.Worksheets.Add("Sheet1");
        sheet1.Cells["E2"].Value = 12;
        sheet1.Cells["E3"].Value = 23;
        sheet1.Cells["E4"].Value = 34;
        sheet1.Cells["E5"].Value = 45;

        sheet1.Cells["C3"].Formula = "E:E";
        sheet1.Cells["C4"].Formula = "E:E";

        sheet1.Cells["C3:C4"].UseImplicitItersection = true;

        pck.Workbook.Calculate();

        Assert.AreEqual(23, sheet1.Cells["C3"].GetValue<double>(), Tolerance);
        Assert.AreEqual(34, sheet1.Cells["C4"].GetValue<double>(), Tolerance);
    }
}
```

**Comment by keithclarkj** [comment]
I have tried changing the formula to using a specified range (i.e. `E2:E5`) and this works - seems this has something to do with the whole column being used.

**Comment by JanKallman** [comment]
Yes, this seems to be an issue when you use full column / row addresses. I will look at it and will provide a fix.

**Comment by JanKallman** [comment]
Fixed in 7.0.8
